### ISSUE TITLE
SelectOneMenu misuses aria attribute on <input>

### ISSUE THREAD

**Issue by MarkusChur** [primefaces/primefaces#4832]
SelectOneMenu generates text fields for autocomplete/filtering that have aria-expanded attribute set. Accessibility tools classify this as failure, e. g. chrome addon: Accessibility Insights for Web.
By removing this attribute on inputs the source seems to be WCAG compliant.

https://www.w3.org/WAI/WCAG21/Understanding/parsing.html
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

https://github.com/primefaces/primefaces/blob/9208733aa9f10483256f93d8a79815ffe774d3d7/src/main/java/org/primefaces/component/selectonemenu/SelectOneMenuRenderer.java#L163

**Comment by melloware** [comment]
PR Submitted. Thanks for reporting!
